src/ParseServer.js

```diff
@@ -80,6 +81,9 @@ class ParseServer {
         if (serverStartComplete) {
           serverStartComplete();
         }
+        if ((options.securityChecks || {}).enableLogOutput) {
+          this.getSecurityChecks();
+        }
```

Comment:
Run security check after serverStartComplete promise resolved (if serverStartComplete return a promise) to avoid false/positive, since developers can inject some initialization script into serverStartComplete 😉